### DISCUSSION TITLE
Regresión o clasificación

### DISCUSSION THREAD

**Discussion by Joanri20** [rramosp/ai4eng.v1#267] [01 - INTRODUCCIÓN]
https://youtu.be/8mp9T1v9ezc?t=908

En el video, habla de un problema de regresión y clasificación, podria por favor  explicarme los terminos a que se refiere.

**Comment by rramosp** [comment]
las tareas 1 y 2 de ese laboratorios son de clasificación (la predicción
sólo puede ser 0 o 1)

la tarea 3 es de regresión (la predicción puede ser cualquier valor >0)

On Sat, Aug 26, 2023 at 7:37 PM Joan Rivas ***@***.***> wrote:

> https://youtu.be/8mp9T1v9ezc?t=908
>
> En el video, habla de un problema de regresión y clasificación, podria por
> favor explicarme los terminos a que se refiere.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/267>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIMZQ4WVNJXKY4MOQY3XXKJE3ANCNFSM6AAAAAA375TFZQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
